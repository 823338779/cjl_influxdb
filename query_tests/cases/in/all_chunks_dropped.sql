-- IOX_SETUP: OneMeasurementAllChunksDropped

-- list information schema (show that all the chunks were dropped)
SELECT * from information_schema.tables where table_schema = 'iox';
