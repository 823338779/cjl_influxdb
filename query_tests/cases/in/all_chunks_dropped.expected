-- Test Setup: OneMeasurementAllChunksDropped
-- SQL: SELECT * from information_schema.tables where table_schema = 'iox';
+---------------+--------------+------------+------------+
| table_catalog | table_schema | table_name | table_type |
+---------------+--------------+------------+------------+
| public        | iox          | h2o        | BASE TABLE |
+---------------+--------------+------------+------------+
